docs/developing_clusterfuzzlite.md

```diff
@@ -13,13 +13,6 @@ permalink: /developing-clusterfuzzlite/
 {:toc}
 ---
 
-TODO
```

Comment:
Can you leave these for now? I'm going to fix them today.

Reply:
Reinstated.